src/main/java/org/cryptomator/common/settings/SettingsProvider.java

```diff
@@ -58,7 +62,13 @@ public Settings get() {
 	}
 
 	private Settings load() {
-		Settings settings = env.getSettingsPath().flatMap(this::tryLoad).findFirst().orElseGet(() -> Settings.create(env));
+		Settings settings = env.getSettingsPath() //
+				.flatMap(this::tryLoad) //
+				.findFirst() //
+				.orElseGet(() -> Settings.create(env));
+		if (settings.quickAccessService.getValue() == null) {
```

Comment:
mark this for removal, as we shouldn't carry around data migration code for too long. If late adopters update from an old version after we removed the migration, there is not much harm if they need to enable this manually.

Reply:
This is no migration code, but actual app logic.

The feature should be activated by default. To achieve this, we need not only set the `useQuickAccess` property, but also select a `quickAccessService`. Using the strategy "use the one with highest priority if the value is null", is not a good approach, since then the app behaviour can suddenly change without user interaction.
We cannot use the preferences window to set this option, because then a user has to open the preferences window to actually use the feature. Also, I did not followed the keychain approach by hardcoding a service provider name.

The current approach allows to always have a valid service selected, if one is available and the user did not chose one before. If the selected service is not available anymore, then nothing is done (as expected).

Reply:
After a discussion. this approach is dropped:
1. Settings are loaded at app start. If the settings require service provider, these must also be loaded and this could (potentially/ in the future) take a long time, delaying the app start by a good proportion
2. We cannot differ between the user selected the quickAccess impl and it was selected by default at some point in time. Settings migration at a later point in time would have to deal with not only "null" but also all implementations which could have been selected and we don't no if this was an active user choice.

i implemented the "hard coded" solution.